packages/viewer/src/components/viewer-markup-arrow/utils.ts

```diff
@@ -0,0 +1,56 @@
+import { Angle, Matrix, Point } from '@vertexvis/geometry';
+
+export interface ArrowheadPoints {
+  tip: Point.Point;
+  left: Point.Point;
+  right: Point.Point;
+  base: Point.Point;
+}
+
+export function createArrowheadPoints(
+  start: Point.Point,
+  end: Point.Point,
+  arrowAngle = 60
+): ArrowheadPoints {
+  const distance = Point.distance(start, end);
+  const angle = Angle.normalize(
+    Angle.toDegrees(Angle.fromPoints(start, end)) - 270
+  );
+
+  const height = Math.max(4, Math.min(16, distance * 0.25));
+  const sideLength = height / Math.cos(Angle.toRadians(90 - arrowAngle));
+
+  const rotation = Matrix.rotation(angle);
+  const arrowLeft = Point.polar(sideLength, Angle.toRadians(arrowAngle * 2));
+  const arrowRight = Point.polar(sideLength, Angle.toRadians(arrowAngle));
+  const arrowBase = Point.polar(height, Angle.toRadians(90)); //  * 0.8
```

Comment:
is this comment needed?

Reply:
Nope! This is pulled over from mfg-ui. I'll remove this